examples/sac/sac.py

```diff
@@ -77,11 +60,11 @@ def make_args():
     "humanoid": 100,
 }
 
-
-def main(args):
+@hydra.main(config_name="config")
+def main(cfg: DictConfig):
     from torch.utils.tensorboard import SummaryWriter
 
-    args = correct_for_frame_skip(args)
+    args = correct_for_frame_skip(cfg)
```

Comment:
since we are replacing argparse, could we also update the variable names to be `cfg` to make it more consistent?